Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -871,6 +871,9 @@ - (void)enterFullScreen:(CGSize)videoDimensions
     page->setSuppressVisibilityUpdates(true);
 
     _viewState.store(webView.get());
+#if PLATFORM(VISION)
```

Comment:
In the interest of having less platform-specific code, would it be harmful to do this on other platforms?

Reply:
Good idea, it shouldn't be harmful, I will confirm.

I am also interested if anyone knows why it does not seem to be necessary on iPad.

Reply:
Confirmed it is fine on iOS and actually I think the issue is present there, but we perform another layout and that corrects the layout.

I removed the change from `PLATFORM(VISION)`